### PR TITLE
Add info on how to create an issue from a discussion

### LINKED ISSUE #8818: Add "Create issue from discussion" information to docs
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's Code of Conduct.

### What article on docs.github.com is affected?

I think the whole Discussions thing itself is still in _beta_, so I don't know if I should expect this to be documented in full???

I would (normally) expect a section on "Creating an issue from a discussion to be in this article:

https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue

### What part(s) of the article would you like to see updated?

Add section `Creating an issue from a discussion`.
Add this section heading to the article's sidebar
Add picture of the button/interface to the article

Explain workflow to Create issue from a discussion (I think clicking the button is enough).
Explain the rights needed (I think you need at least `triage` level rights, but I'm not 100% on this!)

### Additional information

See this screenshot with the arrow to see what button I'm talking about. 😉 

screenshot_create_issue_from_discussion_button

### PR DESCRIPTION
**Pull request by thispsj**
### Why:

Closes #8818 

### What's being changed:

Adds the information about creating an issue from discussion. Also adds a link to this article in Managing Discussions.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@thispsj Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ shortTitle: Manage discussions
...
+As a discussions maintainer, you can create community resources to encourage discussions that are aligned with the overall project goal and maintain a friendly open forum for collaborators. Creating a code of conduct or contribution guidelines for collaborators to follow will help facilitate a collaborative and productive forum. For more information on creating community resources, see "[Adding a code of conduct to your project](/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project)," and "[Setting guidelines for repository contributors](/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)." You can even create an issue from a discussion. For more information, see "[Creating an issue](/issues/tracking-your-work-with-issues/creating-an-issue)."
```
Comment (lecoursen) [inline content/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository.md:17]:
```suggestion
As a discussions maintainer, you can create community resources to encourage discussions that are aligned with the overall project goal and maintain a friendly open forum for collaborators. Creating a code of conduct or contribution guidelines for collaborators to follow will help facilitate a collaborative and productive forum. For more information on creating community resources, see "[Adding a code of conduct to your project](/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project)," and "[Setting guidelines for repository contributors](/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)." 

When a discussion yields an idea or bug that is ready to be worked on, you can create a new issue from a discussion. For more information, see "[Creating an issue](/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-discussion)."
```

File: content/issues/tracking-your-work-with-issues/creating-an-issue.md [hunk 2]
```diff
@@ -97,6 +97,28 @@ You can open a new issue from a specific line or lines of code in a file or pull
 {% data reusables.repositories.assign-an-issue-as-project-maintainer %}
 {% data reusables.repositories.submit-new-issue %}
 
+{% ifversion fpt %}
+
+## Creating an issue from discussion
+
+You can even create an issue from a discussion. Anyone with read access to the repository can create an issue from a discussion. Follow the steps below to create an issue from discussion :
```
Comment (lecoursen) [inline content/issues/tracking-your-work-with-issues/creating-an-issue.md:104]:
```suggestion
People with triage permission to a repository can create an issue from a discussion.

When you create an issue from a discussion, the contents of the discussion post will be automatically included in the issue body, and any labels will be retained. Creating an issue from a discussion does not convert the discussion to an issue or delete the existing discussion. For more information about {% data variables.product.prodname_discussions %}, see "[About discussions](/discussions/collaborating-with-your-community-using-discussions/about-discussions)."
```

File: content/issues/tracking-your-work-with-issues/creating-an-issue.md [hunk 3]
```diff
@@ -97,6 +97,28 @@ You can open a new issue from a specific line or lines of code in a file or pull
...
+   ![Create Issue from Discussion](/assets/images/help/discussions/create-issue-from-discussion.jpg)
+{% data reusables.repositories.type-issue-title-and-description %}
+{% data reusables.repositories.assign-an-issue-as-project-maintainer %}
+{% data reusables.repositories.submit-new-issue %}
+
+{% note %}
+
+**Note**: Creating an issue from a discussion **does not** convert the discussion to an issue or delete the existing discussion. Only a new issue is created and the discussion from which it was created is linked to it. For more information, see "[About Discussions](/discussions/collaborating-with-your-community-using-discussions/about-discussions)" and "[Moderating Discussions](/discussions/managing-discussions-for-your-community/moderating-discussions)".
+
+{% endnote %}
```
Comment (lecoursen) [inline content/issues/tracking-your-work-with-issues/creating-an-issue.md:118]:
```suggestion
```

**Review by lecoursen — COMMENTED** [review]
Thanks @thispsj! I am going to apply some changes to better follow our style guide and content model and then I'll merge!

**Comment by lecoursen** [comment]
👋🏻 @thispsj, actually, it looks like I'm not able to apply my suggestions because I don't have edit access to the branch. Can you check the box to allow edits from maintainers in the right sidebar? 🙏🏻

File: content/issues/tracking-your-work-with-issues/creating-an-issue.md (hunk 2 above)
Comment (thispsj) [reply content/issues/tracking-your-work-with-issues/creating-an-issue.md:104]: @lecoursen I think even people with read access can do so.

See even I can do that in github/docs :

Create Issue from Discussion

I guess read access ≠ triage access

Comment (lecoursen) [reply content/issues/tracking-your-work-with-issues/creating-an-issue.md:104]: Thanks! Our internal issue for this feature specifically mentions triage. Maybe we misunderstood, though! I'll double check with that team. 👍🏻

**Comment by lecoursen** [comment]
@thispsj, I saw you 👍🏻 reacted to my last comment but I'm still not able to apply my suggestions. 😕 Where you able to find and check that box? Folks often have trouble with it for some reason. Thanks!

**Comment by thispsj** [comment]
> @thispsj, I saw you 👍🏻 reacted to my last comment but I'm still not able to apply my suggestions. 😕 Where you able to find and check that box? Folks often have trouble with it for some reason. Thanks!

@lecoursen really sorry, I just forgot to enable that.

Anyways I'll pull these myself now

File: content/issues/tracking-your-work-with-issues/creating-an-issue.md (hunk 2 above)
Comment (thispsj) [reply content/issues/tracking-your-work-with-issues/creating-an-issue.md:104]: @lecoursen I will take these in after you've confirmed with the team

**Review by lecoursen — APPROVED** [review]

**Pull request merged**